### DISCUSSION TITLE
Flag "Abilita invio al Sistema Tessera Sanitaria" - Sistema TS

### DISCUSSION THREAD

**Discussion by Caine85** [fattureincloud/api#278] [Q&A] [answered]
Salve a tutti,
è possibile durante la creazione di una fattura tramite API abilitare anche il flag "Abilita invio al Sistema Tessera Sanitaria"?

Ho notato che di default non viene attivato tramite API, al contrario di quanto accade durante la creazione manuale di una fattura.

Grazie mille.

**Comment by Caine85** [comment, marked as answer]
Ho trovato la risposta alla domanda...purtroppo però non posso cancellare la discussione.

Riporto un'immagine che chiarisce dove trovare questi flag:
![219424689-4065e884-d54a-46de-8930-17fb7711c0a1](https://user-images.githubusercontent.com/5676445/219434514-1c9ca514-79d5-489d-a42e-93f4ac7c9f21.png)

**Reply by valmoz** [reply]
Va benissimo così, servirà anche per altri utenti con lo stesso problema.

Grazie
Mauro
